### PR TITLE
chore: add the guide for self-host translations [skip ci]

### PR DESCRIPTION
**Pull request by Jinjiang**
@posva for translations hosted in a different repo, I added more content in the `contributing.md`. Do you think that makes sense? Thanks.

### CODE REVIEW

File: .github/contributing.md [hunk 1]
```diff
@@ -117,6 +117,10 @@ Currently, all the docs can be found in `packages/docs`. It contains the English
 
 Besides that, the `.vitepress` sub-folder is used to put the config and theme, including the i18n information.
 
+For contributing to the translations, you have 2 ways to go:
```
Comment (posva) [inline .github/contributing.md:120]:
```suggestion
Consider the following two options in order to contribute to the translations:
```

File: .github/contributing.md [hunk 2]
```diff
@@ -139,6 +143,17 @@ If you want to maintain a existing translation:
 
 For more real examples, please check out [all the PRs with title "docs(zh): sync" after 2023-01-01](https://github.com/vuejs/router/pulls?q=is%3Apr+created%3A%3E2023-01-01+docs%28zh%29+sync).
 
+### Self-host the translation
+
+You can also host the translation on your own. To create one, just simply fork our GitHub repo and change the content and site config in `packages/docs`. To long-term maintain it, we _highly recommend_ a similar way that we do above for our officially hosted translations:
+
+1. Make sure to have a _checkpoint_ branch (named like `sync`), which is always to the commit of the original docs that your latest translation is corresponding to.
```
Comment (posva) [inline .github/contributing.md:150]:
```suggestion
1. Ensure you create a _checkpoint branch_ (for example, a branch named sync). This branch should always align with the commit of the original documentation that corresponds to your most recent translation.
```

File: .github/contributing.md [hunk 3]
```diff
@@ -139,6 +143,17 @@ If you want to maintain a existing translation:
...
+2. Translate by the diff result between the latest official repo and yours via git diff command or GitHub Compare page.
+3. Finish the translation.
+4. Update the checkpoint branch.
```
Comment (posva) [inline .github/contributing.md:153]:
```suggestion
2. Utilize the diff result between the latest official repository and your own by using the git diff command or the GitHub Compare page to guide your translation.
3. Complete the translation process.
4. Update the _checkpoint branch_ accordingly.
```

**Review by posva — COMMENTED** [review]
Thanks! I tried improving the wording a bit. Let me know what you think!

**Comment by Jinjiang** [comment]
@posva all look great. Updated as well. Thanks.

**Pull request merged**